toolchain/flags.cmake

```diff
@@ -2,11 +2,11 @@
 
 set(
   CMAKE_C_FLAGS
-  "-ffreestanding -nostdlib -nostdinc -fdiagnostics-color=always -fno-common ${CMAKE_C_FLAGS}"
+  "-ffreestanding -nostdlib -nostdinc -fdiagnostics-color=always -fno-common -Wno-pointer-arith ${CMAKE_C_FLAGS}"
 )
 set(
   CMAKE_CXX_FLAGS
-  "-ffreestanding -nostdlib -nostdinc -nostdinc++ -fdiagnostics-color=always -fno-common -fno-rtti -fno-exceptions ${CMAKE_CXX_FLAGS}"
+  "-ffreestanding -nostdlib -nostdinc -nostdinc++ -fdiagnostics-color=always -fno-common -fno-rtti -fno-exceptions -Wno-pointer-arith ${CMAKE_CXX_FLAGS}"
```

Comment:
no no no no no

this should go in `cmake/flags.cmake`

this is for base flags needed to get the toolchain to even work